workflow/rules/calling.smk

```diff
@@ -37,8 +37,8 @@ rule varlociraptor_preprocess:
         ref="resources/genome.fasta",
         ref_idx="resources/genome.fasta.fai",
         candidates=get_candidate_calls(),
-        bam=lambda w: get_sample_bam(w),
-        bai=lambda w: get_sample_bam(w, bai=True),
+        bam="results/recal/{sample}.sorted.bam",
+        bai="results/recal/{sample}.sorted.bai",
```

Comment:
No need for the function anymore?

Reply:
Actually no. This had to be removed due to restructuring.
Previously the input bams for `varlociraptor_preprocess` were either the recalibrated bams or alternatively output of primer trimming. So, recalibration happened before trimming.
As there were some incompatibilities between newly introduced steps base recalibration has moved to the end of the preprocessing workflow so the input needed by `varlociraptor_preprocess` will always be the recalibrated one.